### PR TITLE
PLASTIC-19024 add group api documentation

### PR DESCRIPTION
**Pull request by jasonyzw**
## Context
Adding group related endpoints into a `groups-api` collection to make group functionality more discoverable and organized

## Checklist
- [x] Change meets or does not compromise the [Baseline Security Requirements](https://transferwise.atlassian.net/wiki/spaces/EKB/pages/434929973/Baseline+Security+Requirements)

### CODE REVIEW

File: groups-api/Groups API Sandbox (empty).postman_environment.json
Comment (timotheehub) [inline groups-api/Groups API Sandbox (empty).postman_environment.json]: Let's call it `profile_id` to be consistent with the other Postman collection.

File: groups-api/Groups API Sandbox (empty).postman_environment.json
Comment (timotheehub) [inline groups-api/Groups API Sandbox (empty).postman_environment.json]: Same here. Let's call it `group_id`.

File: groups-api/Groups API Sandbox (empty).postman_environment.json
Comment (timotheehub) [inline groups-api/Groups API Sandbox (empty).postman_environment.json:11]: We can also add the refresh token so that we can add the `POST {{host}}/oauth/token` API endpoint.

Comment (jasonyzw) [reply groups-api/Groups API Sandbox (empty).postman_environment.json:11]: Added both refersh and token endpoint for ease of test!

File: groups-api/Groups API Sandbox (empty).postman_environment.json
Comment (jasonyzw) [reply groups-api/Groups API Sandbox (empty).postman_environment.json]: Changed!

File: groups-api/Groups API Sandbox (empty).postman_environment.json
Comment (jasonyzw) [reply groups-api/Groups API Sandbox (empty).postman_environment.json]: Changed!

**Comment by jasonyzw** [comment]
@timotheehub I have addressed the comments, but "Create balance group" is still a problem when I try to test on Sandbox.
Maybe we can re-review and get it approved (if there is no more enhancement).
But not merge until we resolve the endpoint issue

**Comment by jasonyzw** [comment]
@timotheehub have retested on Sandbox and it seems to be working now! 
The PR is ready for re-review!

**Review by timotheehub — APPROVED** [review]

File: groups-api/Groups API.postman_collection.json
Comment (timotheehub) [inline groups-api/Groups API.postman_collection.json:156]: I believe we use `refresh` everywhere else.

**Review by timotheehub — COMMENTED** [review]
We'll need to solve the issue with the feature flag. It's fine to merge first though. We could skip the group feature flag if the token is from the partner.

**Pull request merged**
